crown-court-contribution/src/test/java/uk/gov/justice/laa/crime/contribution/controller/ContributionControllerContributionIntegrationTest.java

```diff
@@ -205,45 +197,49 @@ void givenMaatApiException_whenCalculateContributionIsInvoked_thenInternalServer
         ApiMaatCalculateContributionRequest appealContributionRequest = TestModelDataBuilder.buildAppealContributionRequest();
         String requestData = objectMapper.writeValueAsString(appealContributionRequest);
 
-        var repOrderUrl = UriComponentsBuilder.fromUriString(getRepOrderUrl)
-                .build(appealContributionRequest.getRepId());
-
-
-        wiremock.stubFor(get(repOrderUrl.getPath())
+        var repOrderUrl = "/rep-orders/" + appealContributionRequest.getRepId();
+        ErrorDTO errorDTO = ErrorDTO.builder()
+                .code(HttpStatus.NOT_IMPLEMENTED.name())
+                .build();
+        wiremock.stubFor(get(urlEqualTo(repOrderUrl))
                 .willReturn(
                         WireMock.aResponse()
                                 .withStatus(NOT_IMPLEMENTED.code())
+                                .withBody(Json.write(errorDTO))
                                 .withHeader("Content-Type", String.valueOf(MediaType.APPLICATION_JSON))
                 )
         );
 
         mvc.perform(buildRequestGivenContent(HttpMethod.POST, requestData, ENDPOINT_URL))
-                .andExpect(status().isInternalServerError())
+                .andExpect(status().is5xxServerError())
                 .andExpect(content().contentType(MediaType.APPLICATION_JSON));
     }
 
     @Test
     void givenMaatApiException_whenGetContributionSummariesIsInvoked_thenInternalServerErrorResponse() throws Exception {
-        var summariesUrl = UriComponentsBuilder.fromUriString(summaryUrl)
-                .build(TestModelDataBuilder.REP_ID);
-        wiremock.stubFor(get(summariesUrl.getPath())
+        var summariesUrl = "/contributions/" + TestModelDataBuilder.REP_ID + "/summary";
+        ErrorDTO errorDTO = ErrorDTO.builder()
+                .code(HttpStatus.NOT_IMPLEMENTED.name())
+                .build();
+        wiremock.stubFor(get(urlEqualTo(summariesUrl))
                 .willReturn(
                         WireMock.aResponse()
                                 .withStatus(NOT_IMPLEMENTED.code())
+                                .withBody(Json.write(errorDTO))
                                 .withHeader("Content-Type", String.valueOf(MediaType.APPLICATION_JSON))
                 )
         );
         mvc.perform(buildRequestGivenContent(HttpMethod.GET, "", GET_CONTRIBUTION_SUMMARIES_ENDPOINT_URL))
```

Comment:
It looks like the `buildRequestGivenContent` method is available in `crime-commons-classes` module, can we use the version please? That way we can clean up the `RequestBuilderUtils` class in this service.

Reply:
Done. Updated the PR description to reflect the change.